linux/scripts/launchpad.sh

```diff
@@ -4,8 +4,9 @@
 
 # must be run from linux dir
 
-# parameters: [UPLOAD="yes"] [PROJECT="<project>"] [DIST="<dist>"] ./scripts/launchpad.sh
+# parameters: [UPLOAD="yes"] [TIER="<tier>"] [PROJECT="<project>"] [DIST="<dist>"] [PACKAGEVERSION="<version>"] ./scripts/launchpad.sh
 # UPLOAD="yes" do the dput for real
+# TIER="<tier>" alpha, beta or stable, default beta
```

Comment:
why wouldn't the tier default to `stable`? 

(Granted we're currently in 11.0 beta phase)

Reply:
because we are in the beta phase

Reply:
I think the question is "why would we want to push beta by default, even when we reach stable?"  Since we only have 'beta' available now, doing this temporarily makes sense, but when time comes for 12.0 beta, I'd expect 11.0 stable to be the main push unless someone actually prefers beta.